app/views/admin/tasks/_claim_summary.html.erb

```diff
@@ -49,6 +49,17 @@
         <%= l(claim.submitted_at) %>
       </dd>
     </div>
+
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        Payroll status
```

Comment:
I'm not sure "Payroll status" is the clearest indicator of what this field will tell us - "Awaiting payroll" is certainly a status, but "April 2020" is more of an identifier of which payroll the claim was paid out in.

Perhaps changing the output when a payroll has been processed to something like "Included in April 2020 payroll run"?

Reply:
This is a good point. I can see value in making it clearer and more descriptive.
I had used "April 2020", or rather "Month Year", as this was what Rich had stated within the ticket description points. 
@tekin - do you feel this would be helpful?

Reply:
Based on the limited time we have, I think it will need to be left as it is and could be something to be looked at for future devs